openedx/core/djangoapps/embargo/views.py

```diff
@@ -48,9 +48,9 @@ def get(self, request):
                         response['access'] = False
                         break
                 except InvalidKeyError:
-                    return Response(data=None, status=status.HTTP_400_BAD_REQUEST)
+                    return Response('Invalid course_ids', status=status.HTTP_400_BAD_REQUEST)
```

Comment:
This will be a different format than the other APIs. Raise a ValidationError and set the message on the detail kwarg. http://www.django-rest-framework.org/api-guide/exceptions/#validationerror